pyomo/contrib/fbbt/interval.py

```diff
@@ -74,11 +80,7 @@ def inv(xl, xu, feasibility_tol):
 
 
 def div(xl, xu, yl, yu, feasibility_tol):
-    if xl <= 0 <= xu and yl <= 0 <= yu:
-        lb = -inf
-        ub = inf
-    else:
-        lb, ub = mul(xl, xu, *inv(yl, yu, feasibility_tol))
+    lb, ub = mul(xl, xu, *inv(yl, yu, feasibility_tol))
     return lb, ub
```

Comment:
It is slightly faster to just `return mul(...` (Python doesn't have to unpack the tuple, update the `locals()`, recreate the tuple, and increment / decrement references)